era5_cloud/create_artifact.jl

```diff
@@ -1,83 +1,116 @@
 using NCDatasets
 using ClimaArtifactsHelper
 
-FILE_PATH = "era5_cloud_201001-201012.nc"
-
 const H_EARTH = 7000.0
 const P0 = 1e5
 const HPA_TO_PA = 100.0
 
 Plvl_inv(P) = -H_EARTH * log(P / P0)
 
-function create_cloud(infile_path, outfile_path)
-    ncin = NCDataset(infile_path)
-    ncout = NCDataset(outfile_path, "c")
+short_name_dict = Dict(
+    "fraction_of_cloud_cover" => "cc",
+    "specific_cloud_liquid_water_content" => "clwc",
+    "specific_cloud_ice_water_content" => "ciwc",
+    "specific_humidity" => "q",
+)
+
+function create_cloud(variables, outfile_path; small=false)
 
-    # This artifact is for temporarily prescribing clouds in our simulations, so we
-    # only need a coarse resolution. Here we resample the ERA5 data (0.25x0.25 degrees)
-    # to 2x2 degrees.
-    THINNING_FACTOR = 8
+    ncout = NCDataset(outfile_path, "c")
+    infile_path = map(a -> "era5_cloud_hourly_"*variables[1]*"_2010"*lpad(string(a),2,"0")*".nc", collect(1:12))
```

Comment:
```suggestion
infile_path = map(m -> "era5_cloud_hourly_$(variables[1])_2010$(m).nc", lpad.(1:12, 2, "0"))
```
I'd also call this `infile_paths`

Reply:
Thanks!